packages/core/src/accordion/accordion-panel.element.spec.ts

```diff
@@ -31,10 +31,13 @@ describe('accordion-panel element', () => {
   it('should create the component', async () => {
     await componentIsStable(component);
     const slots = getComponentSlotContent(component);
-    expect(slots['accordion-header'].includes('<cds-accordion-header slot="accordion-header">')).toBe(true);
+    expect(slots['accordion-header'].startsWith('<cds-accordion-header slot="accordion-header"')).toBe(true);
     expect(slots['accordion-header'].includes(placeholderHeader)).toBe(true);
 
-    expect(slots['accordion-content'].includes('<cds-accordion-content slot="accordion-content">')).toBe(true);
+    console.log(slots);
```

Comment:
Do we need the `console.log()` here?

Reply:
nope. I forgot to remove after I fixed the test.

Reply:
removed.